boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -69,10 +64,11 @@ CPP = $(CROSSDEV)cc -E -P -x c
 LD = $(CROSSDEV)ld
 ifeq ($(CONFIG_HOST_MACOS),y)
 STRIP = $(CROSSDEV)strip
+AR = $(TOPDIR)/tools/xcode-silent-ar-rcs
```

Comment:
Should we give the new file .sh suffix?
Change to macar like zdsar?

Reply:
how about macar-rcs.sh then?

Reply:
Fine with me

Reply:
done